src/containers/SendContainer/SendContainer.js

```diff
@@ -90,23 +96,25 @@ const SendContainer = (props: Props) => {
       footer={<SendFooter {...footerProps} />}
       minAvoidHeight={800}
     >
-      <Selector
-        label={t('label.to')}
-        placeholder={t('label.chooseReceiver')}
-        searchPlaceholder={t('label.walletAddress')}
-        wrapperStyle={{ marginTop: spacing.medium }}
-        noOptionImageFallback
-        hasQRScanner
-        disableSelfSelect
-        allowEnteringCustomAddress
-        {...customSelectorProps}
-      />
-      <ValueSelectorCard
-        selectorModalTitle={t('transactions.title.valueSelectorModal')}
-        maxLabel={t('button.sendMax')}
-        wrapperStyle={{ paddingTop: spacing.medium }}
-        {...customValueSelectorProps}
-      />
+      <Wrapper>
+        <ValueSelectorCard
+          selectorModalTitle={t('transactions.title.valueSelectorModal')}
+          maxLabel={t('button.sendMax')}
+          wrapperStyle={{ paddingTop: spacing.medium }}
+          {...customValueSelectorProps}
+        />
+        <ArrowIcon />
+        <Spacing h={16} />
```

Comment:
Since this tends to repeat, why not include it into `Selector` (based on prop? Say, topIconProps? And then if it's present, render an Icon with such props?)

Also, `h={20}` :)

Reply:
I don't think it's worth it. Sometimes the arrow is above the selector, sometimes it's under. And those are just two lines of code. I don't feel like the arrow is a part of the selector, but rather it's a part of the screen and I like to keep components as simple as possible.

Reply:
Alright 👍